### PR TITLE
Issue/882 change background link on homepage

### PR DESCRIPTION
**Pull request by ketikat**
### What this PR does

- Change <a> tag in Lozenge.js component to react-router-dom <Link> tag

### Checklist
- [ ] There are unit tests to verify my changes are correct | Unit tests aren't applicable (delete one)
- [ ] I've updated CHANGES.md with what I changed.
- [ ] I've linked this PR to an issue in ZenHub (or there is no issue) and dragged it to the _Pull Request_ column

### CODE REVIEW

File: magda-web-client/src/Pages/HomePageComponents/StoryBox.js [hunk 1]
```diff
@@ -25,9 +26,9 @@ class StoryBox extends Component {
     getClickableElement(el, url) {
         if (!url) return el;
         return (
-            <a target="_blank" href={url} className="story-title-link">
+            <Link target="_blank" to={url} className="story-title-link">
```
Comment (steve9164) [inline magda-web-client/src/Pages/HomePageComponents/StoryBox.js]: This link is actually the link used for stories, like this one:
image

**Comment by steve9164** [comment]
The `<a>` tag mentioned in #882 is this one: https://sourcegraph.com/github.com/TerriaJS/magda/-/blob/magda-web-client/src/Pages/HomePageComponents/Lozenge.js#L9:1

**Comment by AlexGilleran** [comment]
Thanks so much for raising a PR with us @ketikat! Great to have new contributors :D.

But yeah, as @steve9164 mentioned, the stories have to be able to support external links so I don't think we can change them to `Link` elements without some kind of logic to detect whether they're internal or external first.

**Comment by steve9164** [comment]
No worries @ketikat. Drop into our team chat [on gitter](https://gitter.im/magda-data/Lobby) if you have any questions. And let us know how you go and how we can improve our docs (which are a little scarce for developing just the frontend by the look of it).

**Comment by ketikat** [comment]
@steve9164  Okay, I've updated this PR as per your comments, so Lozenge.js is the file being updated.

**Comment by steve9164** [comment]
Thanks @ketikat. A couple of final things:
- Please add yourself to the contributors list (CONTRIBUTORS.md). Just add a new top-level bullet point with your name (or username) linking to your GitHub account, and
-  Please add a CHANGES.md entry under 0.0.39

**Comment by ketikat** [comment]
@steve9164   good to go I think?

**Comment by steve9164** [comment]
Thanks for the contribution!

**Pull request merged**